src/tests/ftest/util/ior_utils.py

```diff
@@ -135,18 +135,14 @@ def run_ior(client_file, ior_flags, iteration, block_size, transfer_size,
     attach_info_path = basepath + "/install/tmp"
     try:
 
-        ior_cmd = (
-            orterun_bin + " -N {} --hostfile {} -x DAOS_SINGLETON_CLI=1 "
-            "-x CRT_ATTACH_INFO_PATH={} ior {} -s {} -i {} -a DAOS -o {} -b {} "
-            "-t {} -- --daos.pool {} --daos.svcl {} --daos.recordSize {} "
-            "--daos.stripeSize {} --daos.stripeCount {} --daos.aios {} "
-            "--daos.objectClass {} ".format(slots, client_file,
-                                            attach_info_path, ior_flags,
-                                            seg_count, iteration, filename,
-                                            block_size, transfer_size,
-                                            pool_uuid, svc_list, record_size,
-                                            stripe_size, stripe_count, async_io,
-                                            object_class))
+        ior_cmd = orterun_bin + " -N {} --hostfile {} -x DAOS_SINGLETON_CLI=1 "\
```

Comment:
Shouldn't this use `get_ior_cmd()` and the updates to the the ior command should be made in that function?